static/js/actions/commonUi.js

```diff
@@ -1,28 +1,31 @@
 // @flow
-import { createAction } from 'redux-actions';
+import { createAction } from "redux-actions";
 
 export const qualifiedName = (name: string) => `UI_${name}`;
 
-export const SET_DRAWER_OPEN = qualifiedName('SET_DRAWER_OPEN');
+export const SET_DRAWER_OPEN = qualifiedName("SET_DRAWER_OPEN");
 export const setDrawerOpen = createAction(SET_DRAWER_OPEN);
 
-export const SHOW_DIALOG = qualifiedName('SHOW_DIALOG');
+export const SHOW_DIALOG = qualifiedName("SHOW_DIALOG");
 export const showDialog = createAction(SHOW_DIALOG);
 
-export const HIDE_DIALOG = qualifiedName('HIDE_DIALOG');
+export const HIDE_DIALOG = qualifiedName("HIDE_DIALOG");
 export const hideDialog = createAction(HIDE_DIALOG);
 
-export const INIT_EDIT_VIDEO_FORM = qualifiedName('INIT_EDIT_VIDEO_FORM');
+export const INIT_EDIT_VIDEO_FORM = qualifiedName("INIT_EDIT_VIDEO_FORM");
 export const initEditVideoForm = createAction(INIT_EDIT_VIDEO_FORM);
 
-export const SET_EDIT_VIDEO_TITLE = qualifiedName('SET_EDIT_VIDEO_TITLE');
+export const SET_EDIT_VIDEO_TITLE = qualifiedName("SET_EDIT_VIDEO_TITLE");
 export const setEditVideoTitle = createAction(SET_EDIT_VIDEO_TITLE);
 
-export const SET_EDIT_VIDEO_DESC = qualifiedName('SET_EDIT_VIDEO_DESC');
+export const SET_EDIT_VIDEO_DESC = qualifiedName("SET_EDIT_VIDEO_DESC");
 export const setEditVideoDesc = createAction(SET_EDIT_VIDEO_DESC);
 
-export const SHOW_MENU = qualifiedName('SHOW_MENU');
+export const SHOW_MENU = qualifiedName("SHOW_MENU");
 export const showMenu = createAction(SHOW_MENU);
 
-export const HIDE_MENU = qualifiedName('HIDE_MENU');
+export const HIDE_MENU = qualifiedName("HIDE_MENU");
 export const hideMenu = createAction(HIDE_MENU);
+
+export const SHOW_FAQ_ENTRY = qualifiedName("SET_SHOW_FAQ_ENTRY");
+export const showFAQEntry = createAction(SHOW_FAQ_ENTRY);
```

Comment:
Should probably be called `TOGGLE_FAQ_VISIBILITY` and `toggleFAQVisibility` or something like that

Reply:
sure, that makes sense